src/robusta/integrations/mattermost/sender.py

```diff
@@ -11,6 +11,7 @@
     KubernetesDiffBlock,
     ListBlock,
     MarkdownBlock,
+    ScanReportBlock,
     TableBlock,
 )
 from robusta.core.reporting.utils import add_pngs_for_all_svgs
```

Comment:
I think there's a missing import here
`from robusta.core.reporting.consts import EnrichmentAnnotation`

Can you please add it?